lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -223,7 +229,8 @@ def additional_paths_receive=(val)
     end
 
     def default_additional_paths_receive
-      config_get_default('bgp_neighbor_af', 'additional_paths_receive').to_sym
+      ret = config_get_default('bgp_neighbor_af', 'additional_paths_receive')
+      ret.to_sym unless ret.to_s == ''
```

Comment:
Also, it shouldn't be too hard to extend CommandReference to let the YAML specify `kind: symbol` and have CmdRef automatically do this conversion.